runtime/zephir.c

```diff
@@ -108,7 +108,7 @@ static void zephir_initialize_memory(zend_zephir_globals *zephir_globals_ptr TSR
 static void zephir_compile_program(zval *program TSRMLS_DC)
 {
 	HashTable           *ht = Z_ARRVAL_P(program);
-	HashPosition        pos = {0};
+	HashPosition        pos;
```

Comment:
`typedef uint32_t HashPosition` in PHP 7
`typedef Bucket* HashPosition` in PHP 5

I would assign it to plain 0, not sure why `{ 0 }` was used.

Bracketed initialization also works though: http://rextester.com/GXS88393